src/test/java/org/broadinstitute/consent/http/db/AcknowledgementDAOTest.java

```diff
@@ -0,0 +1,75 @@
+package org.broadinstitute.consent.http.db;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.broadinstitute.consent.http.models.Acknowledgement;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static java.lang.Thread.sleep;
+import static org.junit.Assert.*;
+
+public class AcknowledgementDAOTest extends DAOTestHelper {
+    @Test
+    public void createAndRetrieveAcknowledgement() throws InterruptedException {
```

Comment:
tests are a bit long, maybe worth investigating breaking up into multiple tests, just for readability. they look great though

Reply:
Unless there's strong objection, I think I'm going to leave these tests as-is.  The multiple statements and validation cover the upsert behavior of postgres to validate the SQL is being handled as expected.